components/Header/header.scss

```diff
@@ -286,100 +286,100 @@ code {
   .with-mark.js-facet-clear:hover {
     background-position: 95% center;
   }
-  
+
   .dmpui-frontend-btn-secondary, .btn-default, button.dropdown-toggle,
   .c-search-clear,
   .js-facet-clear {
     background-color: #fff;
     color: var(--dmpui-brand-blue-900);
     border: 2px solid var(--dmpui-brand-blue-900);
   }
-  
+
   .dmpui-frontend-btn-secondary:hover, .btn-default:hover, button.dropdown-toggle:hover,
   .c-search-clear:hover,
   .js-facet-clear:hover {
     background-color: var(--dmpui-brand-blue-400);
     border: 2px solid var(--dmpui-brand-blue-400);
   }
-  
+
   .dmpui-frontend-header {
     padding: 1rem 0.7rem;
     display: flex;
     align-items: center;
     gap: 15px;
   }
-  
+
   .dmpui-frontend-header .header-logo img {
     max-width: 260px;
   }
-  
+
   .dmpui-frontend-header ul {
     padding: 0;
     margin: 0 1rem 0 0;
   }
-  
+
   .dmpui-frontend-header ul li {
     display: inline-block;
     margin: 0 0.5rem;
   }
-  
+
   .dmpui-frontend-header ul li a {
     color: var(--dmpui-brand-blue-900);
     text-decoration: none;
     font-weight: 700;
   }
-  
+
   .dmpui-frontend {
-    font-family: var(--font-poppins), sans-serif;
+    font-family: var(--font-inter), sans-serif;
```

Comment:
Changing font variables like this, just because the because the font changed, defeats the point of making the font family a variable. We should define this globally. (See my other comment)